src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  /* background-size: contain; */
+  margin-right: 4px;
+  background-repeat: no-repeat;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--6 .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
check checklist, there is the rule about the case, when you want to set the same property for many classes